polaris-core/src/main/java/org/apache/polaris/core/persistence/transactional/PolarisMetaStoreManagerImpl.java

```diff
@@ -66,6 +66,7 @@
 import org.apache.polaris.core.storage.PolarisStorageActions;
 import org.apache.polaris.core.storage.PolarisStorageConfigurationInfo;
 import org.apache.polaris.core.storage.PolarisStorageIntegration;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Same. I wonder why JetBrains is even present in the classpath? 🤔

Reply:
Will fix this PR locally and look into JetBrains jars separately :+1:

Reply:
Looks like IJ tricks - I override a method with jakarta annotations, but the override gets JetBrains annotations :joy:

Reply:
#1176